buzsaki_lab_to_nwb/tingley_code_septal/tingley_septal_requirements.txt

```diff
@@ -0,0 +1,57 @@
+asciitree==0.3.3
+attrs==21.4.0
+backcall @ file:///home/ktietz/src/ci/backcall_1611930011877/work
+Bottleneck @ file:///tmp/build/80754af9/bottleneck_1607575130224/work
```

Comment:
What are all these @ items? Are they really necessary?

Reply:
They point directly to the build version in the local system which is redundant. Let's go with the minimal pinned version.